src/open_inwoner/kvk/views.py

```diff
@@ -64,6 +64,13 @@ def get(self, request):
             request.session.save()
             return HttpResponseRedirect(redirect.url)
 
+        # create pseudo-branch representing the company as a whole
+        master_branch = {
+            "kvkNummer": company_branches[0]["kvkNummer"],
```

Comment:
Nitpicking, but I would use `request.user.kvk` here, in case the KVK nummer in the API in missing for some reason, but you can leave it as is